### ISSUE TITLE
Instructions for SSO don't work.

### ISSUE THREAD

**Issue by scyto** [MicrosoftDocs/windowsserverdocs#5202]
The configure single sign-on doesn't work.
WAC is terrible to install, its been terrible for years, I try every year or two and generally fail.
its laughable, i have this installed in gateway mode and it won't even connect to itself or any other computer
with AD auth enabled it doesn't work with Windows Hello Auth - if i enter pin too many times i get HTTP 401 error and have to clear browser cache.
the product fundamentally doesn't work 

![image](https://user-images.githubusercontent.com/11369180/104538368-98337600-55d0-11eb-9ee0-a2ca5989961f.png)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bb766b65-907c-1c8e-4af2-fc312a8572ad
* Version Independent ID: a07c9822-76ae-d3cf-5e82-0b56e180d91d
* Content: [Configuring user access control and permissions](https://docs.microsoft.com/en-us/windows-server/manage/windows-admin-center/configure/user-access-control)
* Content Source: [WindowsServerDocs/manage/windows-admin-center/configure/user-access-control.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/manage/windows-admin-center/configure/user-access-control.md)
* Product: **windows-server**
* Technology: **windows-admin-center**
* GitHub Login: @Haley-Rowland
* Microsoft Alias: **harowl**

**Comment by illfated** [comment]
Well, I note your issues and you have my sympathies. That being said, I would like you to know that this repository is purely intended for documentation improvements, not for support issue handling or online help (unless you are lucky enough to have someone drop by and offer you advice on how to solve your issue(s).

Allow me share some useful notes I have accumulated during my time contributing to this repository.

**Comment by illfated** [comment]
Please use a real support instance like Microsoft Support or Microsoft Forums. This repository is intended for documentation improvement only.

Wish I could help, but this repository is not intended for regular support issues, support requests or support questions.

You may want to look elsewhere for answers, unless you have suggestions for improving one or more specific pages in the MS Docs repository.

Disclaimer: I am not affiliated with Microsoft or the Microsoft Docs teams here on GitHub.

The intention of this repository is to improve on the existing documentation by providing the users a way to report factual errors, improvement suggestions and request updates for new versions.

The MS Docs team members might not have time to answer your question, both due to the small size of the team and the big workload.

Try the following resources to get in touch with users who may be able to help you:
- TechNet forums: https://social.technet.microsoft.com/Forums/en-US/home
- MSDN: https://social.msdn.microsoft.com/Forums/en-US/home
- Microsoft Community: https://answers.microsoft.com/

**Comment by scyto** [comment]
I agree it was a bit of rant, that doesn't change the fact the docs are inaccurate and out of date with broken links.

For example following the breadcrumb trail on troubleshooting winrm - it links to articles that reference tools that are no longer available.  The number of blind alleys I had to go down and reject were large.

Where is the one set of instructions about how to setup wirm for HTTPs for WAC - there isn't one  - it's piecemeal.

If I wanted open source docs i would go use free software not purchased software.

The eventual fix.

winrm qc /transport:https

AND

making sure that every host has a 'perfectly' formed certificate - which is interesting because none of the default certs in a Windows CA are.  I am still struggling with that, also there is zero documentation on how to handle cert renewals.  not to mention no documentation on how winrm qc transport:https selects certs - because ta couple of machines absolutely have certs that match the hosts that work - but the command insists there is no valid cert in the machine store....

I have loads more like this.... but i give up and will continue to recommend other products.

This isn't support - it's fundamentally inadequate documentation of a paid product.

**Comment by scyto** [comment]
oh i have an answer to the issue with the good cert not working - seems the machine needs to be rebooted after it auto enrolled. again something it would be useful to say in the documentation  - i don't expect poke'n'hope docs from a paid product.

**Comment by illfated** [comment]
Fair point. @Haley-Rowland , do you think anything from the above comments could be useful to make changes or amendments in the related linked documentation?

**Comment by scyto** [comment]
Maybe i can help?

Here is scenario, i literally can't find any single docs on the web (other than people blocked at piece parts) that is for getting WinRM HTTPs working nicely - this was the root of my issue.  
Anyhoo scenario is:

1) all domain joined servers including WAC gateway sever.
2) client machine is Azure AD domain joined client with full working WhFB signon (with pins and face working)
3) edge browser on the client is with a profile that is signed on with my Azure AD  
4) pre-existing Window Enterprise CA
5) need to make WinRM HTTPs listener refresh its config if the machines cert expires or is invalidated re-issues

Steps to get WinRM HTTPs for WAC fully working:
1) make sure all servers auto enroll for a certificate for server and client auth that in subject name field is CN and nothing else
2) create a GPO policy to open port 5986 on all servers and push to all (because winrim qc creates HTTP firewall policy but not HTTPS firewall policy)
3) create a GPO policy to create a scheduled task that runs a signed powershell script:
```
winrm delete winrm/config/Listener?Address=*+Transport=HTTPS
winrm qc -quiet -transport:https -force
```
the scheduled task needs to be configured to run when it detects
``` 
Path="Microsoft-Windows-CertificateServicesClient-Lifecycle-System/Operational"&gt;&lt;Select Path="Microsoft-Windows-CertificateServicesClient-Lifecycle-System/Operational"&gt;*[System[Provider[@Name='Microsoft-Windows-CertificateServicesClient-Lifecycle-System'] and EventID=1001]
or
Path="Microsoft-Windows-CertificateServicesClient-Lifecycle-System/Operational"&gt;&lt;Select Path="Microsoft-Windows-CertificateServicesClient-Lifecycle-System/Operational"&gt;*[System[Provider[@Name='Microsoft-Windows-CertificateServicesClient-Lifecycle-System'] and EventID=1001]
````
Now i have a fully sustainable HTTPs WinRM infrastructure.  This is multiple days of work.  I don't see anyone who has done and documented this in any blog I can find.

If we genuinely wants folks to use HTTPs then rather than fragments of docs customers  need full how-to's that string multiple disciplines together for real world scenarios.  Each step above was fraught with issues - like why the scheduler wouldn't process the  powershell (because it was unsigned) and then why it wouldn't publish the signed one (because you can't add the CA of all signers to trusted publishers - it has to be the specific signer).  Almost every answer in every technet forum was 'turn off signed execution policy'.

Hilariously this is just me playing in a home-lab (i don't have a windows days job anymore); scrares the crap out of me what people might be doing in the wild.....

To be clear I still don't have SSO to the WAC website from the edge browser:
![image](https://user-images.githubusercontent.com/11369180/104789614-65ad8880-574a-11eb-8b90-7d4ea8b33916.png)

Yet it works perfectly to my Synology DiskStation (SSO using Azure AD on azure joined workstation).

I don't know if this is a configuration error or just WAC not really supporting SSO in the way i expect.
My expectation is my browser autologon to WAC and that when i click the machine i don't get prompted and that when i go to here i would see i am already signed in
![image](https://user-images.githubusercontent.com/11369180/104790014-321f2e00-574b-11eb-9102-64fb1388c62f.png)

also once that is signed on like this
![image](https://user-images.githubusercontent.com/11369180/104790048-4c590c00-574b-11eb-8218-09181746277b.png)

i would expect not to be prompted for creds to individual machines the first time i try and connect to them

Now you might say - setup kerberos constrained delegation (as per the error) i still haven't found a clear doc on that either too.
I have done this (similar to my working kerberos constrained delegation for hyper-v).  But no dice.  I expect this isn't what the error message wants me to do.... 

![image](https://user-images.githubusercontent.com/11369180/104790657-b8883f80-574c-11eb-99ff-21f8119d8f11.png)

**Comment by scyto** [comment]
edit. or i just realized maybe when it says 'logon with windows credential` it doesn't mean my Azure cred?

**Comment by illfated** [comment]
> edit. or i just realized maybe when it says 'logon with windows credential` it doesn't mean my Azure cred?

Not sure about that, but I usually take the term "Windows credential" to mean the local Windows logon information. I don't know the factual answer to this, since I don't know Windows Admin Center well enough to be able to say for certain what the correct information is in this case. Maybe there is some requirement for the local account to be a member of some specific security group. Not even sure about that. That is why I asked for feedback from @Haley-Rowland or anyone else on the MS Docs team.

**Comment by scyto** [comment]
yes me too about local - this is my local logon credential *is* my Azure AD WhFB cred.... that's what confused me.

Either way it should work. 

To be clear this is the scenario i expect to work https://docs.microsoft.com/en-us/windows/security/identity-protection/hello-for-business/hello-how-it-works-authentication#hybrid-azure-ad-join-authentication-using-a-key - i use this for my Azure AD joined (not regular AD DS or Azure AD DS) client to auth to my synology SMB shares, windows server shares and websites (it wont work with RDP, boo).  

I was expecting windows admin center to support the same auth flow (especially as it says it does AAD SSO) - i set the kerberos delegation as suggested by the UI - but no cigar.   Again i don't know if this is a product issue (aka it will never work) or documentation issue (aka nothing to tell me how to make it work).

**Comment by scyto** [comment]
re-opening - request is now for documentation to document fully exactly what SSO scenarios do and don't work.

**Comment by scyto** [comment]
re-opening - request is now for documentation to document fully exactly what SSO scenarios do and don't work.

**Comment by illfated** [comment]
Fair enough. I can only ask for the MS Docs team to provide feedback and relay the information to the relevant authors.

**Comment by robinharwood** [comment]
We hugely appreciate your contribution, feedback, and support to our technical documentation.

We’ve phased out GitHub Issues on this repository as the feedback mechanism for content and have replaced it with a new feedback system. For more information see, [Content User Feedback](https://aka.ms/ContentUserFeedback). As part of this rollout, we are migrating your issue where our team will use your comments to improve the documentation experience. Unfortunately, this does mean you won't be able to access this issue after the migration is complete.

All issues will be migrated by Thursday 29th August 2024.

Once again, thank you for your valuable feedback, we hope you’ll continue to contribute using our new feedback system.
